ice40/regressions/Makefile

```diff
@@ -0,0 +1,8 @@
+NPNR = ../../nextpnr-ice40
+JSON = $(wildcard */*.json)
+OUTPUT = $(patsubst %.json,%.log,$(JSON))
+
+all: $(NPNR) $(OUTPUT)
```

Comment:
I don't think NPNR should be a prereq here - it's not like this Makefile knows how to build it, and this prevents you from doing `make NPNR=nextpnr-ice40`, you have to do `make NPNR=$(which nextpnr-ice40)` instead.

Reply:
You're right: it doesn't know how to make it.
I do however want it to rebuild if the NPNR in my sandbox changed. Whilst I think of how to achieve both, can you think of a solution?